src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,145 @@
+    private final static String UPDATE_CREWMEMBER_SQL = "UPDATE crew_members SET first_name =?, last_name = ?, position = ?, birthday = ?, citizenship = ? WHERE id = ?;";
+    private DataSource dataSource;
+
+    public CrewMemberDaoImpl(DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    @Override
+    public void save(CrewMember crewMember) {
+        try (Connection connection = dataSource.getConnection()) {
+            saveCrewMember(crewMember, connection);
+        } catch (SQLException e) {
+            throw new DaoOperationException(e.getMessage(), e);
+        }
+    }
+
+    private void saveCrewMember(CrewMember crewMember, Connection connection) throws SQLException {
+        PreparedStatement insertStatement = prepareInsertStatement(connection, crewMember);
+        executeUpdate(insertStatement, "Account was not created");
+        int id = fetchGeneratedId(insertStatement);
+        CrewMember.builder().withId(id).build();
+
+    }
+
+    private PreparedStatement prepareInsertStatement(Connection connection, CrewMember crewMember) {
+        try {
+            PreparedStatement insertStatement = connection.prepareStatement(INSERT_CREWMEMBER_SQL, PreparedStatement.RETURN_GENERATED_KEYS);
+            return fillStatementWithAccountData(insertStatement, crewMember);
+        } catch (SQLException e) {
+            throw new DaoOperationException("Cannot prepare statement to insert account", e);
+        }
+    }
+
+    private PreparedStatement fillStatementWithAccountData(PreparedStatement insertStatement, CrewMember crewMember)
+            throws SQLException {
+        insertStatement.setString(1, crewMember.getFirsName());
+        insertStatement.setString(2, crewMember.getLastName());
+        insertStatement.setString(3, String.valueOf(crewMember.getPosition()));
+        insertStatement.setDate(4, Date.valueOf(crewMember.getBirthday()));
+        insertStatement.setString(5, String.valueOf(crewMember.getCitizenship()));
+        return insertStatement;
+    }
+
+    private void executeUpdate(PreparedStatement insertStatement, String errorMessage) throws SQLException {
+        int rowsAffected = insertStatement.executeUpdate();
+        if (rowsAffected == 0) {
+            throw new DaoOperationException(errorMessage);
+        }
+    }
+
+    private int fetchGeneratedId(PreparedStatement insertStatement) throws SQLException {
+        ResultSet generatedKeys = insertStatement.getGeneratedKeys();
+
+        if (generatedKeys.next()) {
+            return generatedKeys.getInt(1);
+        } else {
+            throw new DaoOperationException("Can not obtain an account ID");
+        }
+    }
+
+    public CrewMember findOne(int id) {
+        try (Connection connection = dataSource.getConnection()) {
+            return findCrewMemberById(id, connection);
+        } catch (SQLException e) {
+            throw new DaoOperationException(String.format("Cannot find Account by id = %d", id), e);
```

Comment:
Account -> crew member

And please use one approach for the symmetry principle - or "id" or "ID"